<img src="https://userweb.fct.unl.pt/~jmc.xavier/MSI/iLogos/nsst.jpg" width="400" height="100">

# DEMI | NOVA SST

## Mecânica dos Sólidos I

## Tração e compressão de barras elásticas
## Vigas sujeitas à tracção/compressão. Parte I
## Problema 3 <a name="PIV83"></a>

A figura representa uma barra rígida $BCE$ ligada a dois tirantes $AB$ e $CD$ de aço ($E$ = 200 GPa). Os tirantes estão ligados por articulações nas extremidades e têm secção transversal uniforme com 6$\times$25 mm$^2$. Determine o valor da maior carga $P$ que pode ser suspensa no ponto $E$ sem que o deslocamento vertical nesse ponto exceda 0,25 mm.

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MSI/Notebooks/Au08/Au08_P003_i1.jpg"
width="350" height="220">



### Hipótese
Qual a necessidade de se considerar nesta fase a barra $BCE$ rígida (não deformável e portanto com movimento de corpo rígido)? Na prática esta barra irá estar submetida a uma solicitação de flexão. Contudo, este tópico será apenas oportunamente abordado em Mecânica dos Sólidos II.

Nesta fase, serão usados sólidos designados por barras retilíneas, por hipótese submetidos apenas a esforços axiais ao longo do seu eixo longitudinal.

## Resolução

###### Estratégia de resolução

- Considerar o diagrama de corpo livre da barra rígida ($BCE$) (esta não se deforma, tendo apenas movimentos de corpo rígido).

- Conhecendo as forças em dada tirante, bem como as suas propriedades mecânicas ($E$) e geométricas ($A,L$), os deslocamentos nos pontos extremos dos elementos podem ser determinados pela fórmula de deflexão: $\delta = PL/AE$.

- A deflexão no bordo livro (ponto $E$) onde atua a força $P$, pode ser finalmente usada para a solução do problema por análise da deformada geométrica da barrra rígida; nota: esta barra tem movimento de rotação em torno de um ponto na barra a determianr.

**Diagrama de corpo livre da barra $\mathrm{BCE}$**

Considere-se o diagrama de corpo rígido da barra $\mathrm{BCE}$ com a representação das forças $F_\mathrm{AB}$, $F_\mathrm{CD}$ e $P$ nele aplicadas. O sentido arbitrado para as forças $F_\mathrm{AB}$ e $F_\mathrm{CD}$ preserva o sentido físico da  análise do problema. Isto é, no movimento vertical da barra rígida pela aplicação da força $P$ no ponto $\mathrm{E}$, o sentido indicado para as forças que atuam nos pontos $\mathrm{B}$ e $\mathrm{C}$ sugerem o efeito das barras $AB$ e $DC$ sobre esta (as barras AB e CD estarão à tração pelo que exercem na barra rígida BCE uma força de igual valor mas sentido oposto).

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MSI/Notebooks/Au08/Au08_P003_i2.jpg"
width="350" height="150">

Da análise do equilíbrio estático pode-se concluir ($L = 625$ mm, $l = 250$ mm):

\begin{equation*}
\begin{cases}
\sum F_y = 0 \\[1ex]
\sum M_B = 0 \\
\end{cases}
\quad\Leftrightarrow\quad
\begin{cases}
-F_\mathrm{AB} + F_\mathrm{CD} - P = 0\\[1ex]
F_\mathrm{CD} l - PL = 0 \\
\end{cases}
\quad\Leftrightarrow\quad
\begin{cases}
F_\mathrm{AB} = F_\mathrm{CD} - P
= \left(\displaystyle\frac{L}{l}-1 \right) P = 1.5 P\quad\textrm{(N)} \\[1ex]
F_\mathrm{CD} = \left(\displaystyle\frac{L}{l}\right) P
= 2.5 P\quad\textrm{(N)}\\
\end{cases}
\end{equation*}

De notar que o problema é estaticamente indeterminado, pois existem mais incognitas (3 esforços) do que equações de equilíbrio estático (2 equações). Para a solução do problema é necessário acrescentar uma equação adicional. Em alguns casos esta pode ser obtida por uma equação de compatibilidade de deslocamentos pela análise de uma estrutura dita isostática equivalente. Noutros casos, como o presente exercício, tira-se partido da hipotese de barra rígida indeformável, para estabelecer relações adicionais entre deslocamentos de secções relevantes para a análise do problema.

**Diagrama de corpo livre das barras $\mathrm{AB}$ e $\mathrm{CD}$**

Conside-se agora o diagrama de corpo livre das barras $\mathrm{AB}$ e $\mathrm{CD}$. As duas barras estão à tração, pelo efeito da força externa $P$ aplicada no ponto $E$.

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MSI/Notebooks/Au08/Au08_P003_i3.jpg"
width="350" height="250">

###### Relações deslocamento axial - força axial num elemento retilineo de secção tranversal consante e uniforme.

Considere-se agora a análise da extensão dos pontos $\mathrm{B}$ e $\mathrm{C}$ das barras verticais.

- Ponto $\mathrm{B}$:

\begin{equation*}
\delta_\mathrm{B} = \frac{L_\mathrm{AB}}{A_\mathrm{AB} E} F_\mathrm{AB}
= \frac{ L_\mathrm{AB}}{A_\mathrm{AB} E} \left(\frac{L}{l}-1 \right) P
= 10^{-5} P \quad\textrm{(mm)}
\end{equation*}

In [5]:
import numpy as np

# unidades: mm, N, MPa

E = 200e3 # unidade: MPa
A = 6*25 # unidade: mm^2

LAB = 200
LCD = 200
LBC = 250
LCE = 375
LBE = LBC + LCE

codeltaB = (LBE/LBC-1)*LAB/A/E
print('codeltaB =','{:.2e}'.format(codeltaB))

codeltaB = 1.00e-05


- Ponto $C$:

\begin{equation*}
\delta_\mathrm{C} = \frac{ L_\mathrm{CD}}{A_\mathrm{CD} E} F_\mathrm{CD}
= \frac{ L_\mathrm{CD}}{A_\mathrm{CD} E} \left(\frac{L}{l}\right) P
= 1.667\times10^{-5} P \quad\textrm{(mm)}
\end{equation*}

In [6]:
codeltaC = (LBE/LBC)*LCD/A/E
print('codeltaC =','{:.3e}'.format(codeltaC))

codeltaC = 1.667e-05


**Deformada geométrica (rotação em torno de um ponto) da barra $\mathrm{BCE}$**

- Ponto $E$:

Considere-se o seguinte diagrama da deformada da barra rígida $\mathrm{BCE}$:

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MSI/Notebooks/Au08/Au08_P003_i4.jpg"
width="350" height="240">

Da análise do diagrama da deformada da barra $BCE$ conclui-se, pela igualdade de triângulos:

\begin{equation*}
\tan\alpha = \displaystyle\frac{\delta_\mathrm{B}}{x} = \displaystyle\frac{\delta_\mathrm{C}}{l-x}
\quad\Leftrightarrow\quad
\displaystyle\frac{\delta_\mathrm{B}}{\delta_\mathrm{C}} =
\displaystyle\frac{x}{l-x}
\end{equation*}

Atendendo também que o rácio de variações de comprimento $\delta_\mathrm{B}/\delta_\mathrm{C}$ pode ser expresso por:

\begin{equation*}
\displaystyle\frac{\delta_\mathrm{B}}{\delta_\mathrm{C}} =
\displaystyle\frac{F_\mathrm{AB}L_\mathrm{AB}/A_\mathrm{AB} E}
{F_\mathrm{CD}L_\mathrm{CD}/A_\mathrm{CD} E}
\end{equation*}

uma vez que $L_\mathrm{AB}=L_\mathrm{CD}$ e $A_\mathrm{AB}=A_\mathrm{CD}$, a expressão anterior simplifica-se para:

\begin{equation*}
\displaystyle\frac{\delta_\mathrm{B}}{\delta_\mathrm{C}}
= \displaystyle\frac{F_\mathrm{AB}}{F_\mathrm{CD}}
= \displaystyle\frac{1.5 P}{2.5 P}
= 0.6
\end{equation*}

resultando,

\begin{equation*}
\frac{\delta_\mathrm{B}}{\delta_\mathrm{C}} =
\displaystyle\frac{x}{l-x} = 0.6
\quad\Leftrightarrow\quad
x = \frac{0.6}{1.6}\,l = 0.374~l  \quad\textrm{(mm)}
\end{equation*}

In [8]:
n1 = LBE/LBC -1
d1 = LBE/LBC
dBdC = n1/d1
x = dBdC/(1+dBdC)*LBC
print('x = ','{:.3f}'.format(x),' mm')

x =  93.750  mm


Pela semelhança dos triângulos vem ainda:

- $\delta_B,~x,~\delta_E ~\longrightarrow~P$ :

\begin{equation*}
\tan\alpha = \displaystyle\frac{\delta_\mathrm{B}}{x} =
\displaystyle\frac{\delta_\mathrm{E}}{L-x}~,
\quad \delta_\mathrm{E} \leq 0.25~\textrm{mm}~,
\quad x =  93.75~\textrm{mm}~,
\quad \delta_\mathrm{B} = 10^{-5} P
\end{equation*}

de onde resulta:

\begin{equation*}
P \leq 25000 \frac{x}{L-x}
\end{equation*}

In [9]:
dE = 0.25
P = dE/codeltaB*(x/(LBE-x))
print('P <= ','{:.3f}'.format(P),' N')

P <=  4411.765  N


- $\delta_C,~x,~\delta_E ~\longrightarrow~P$ :

\begin{equation*}
\tan\alpha = \displaystyle\frac{\delta_\mathrm{C}}{l-x} =
\displaystyle\frac{\delta_\mathrm{E}}{L-x}~,\quad \delta_\mathrm{E} \leq 0.25~\textrm{mm}~,
\quad x =  93.75~\textrm{mm}~,
\quad \delta_\mathrm{C} = 1.667e-05 P
\end{equation*}

de onde resulta:

\begin{equation*}
P \leq 15000 \frac{l-x}{L-x}
\end{equation*}

In [10]:
P = dE/codeltaC*((LBC-x)/(LBE-x))
print('P <= ','{:.3f}'.format(P),' N')


P <=  4411.765  N


#### Referências

[1] Beer, Johnston, DeWolf, Mazurek - Mechanics of Materials. 7ª Edição (2014). Problema resolvido 2.1

---

Copyright (c) [Mecânica dos Sólidos I - DEMI - NOVA-SST]

Interactive computing by <a href="https://jupyter.org/" target="_blank"> <span
style="color:#333399"> Jupyter Notebook </span> </a> &nbsp;|&nbsp;Coded by <a href = "mailto: jmc.xavier@fct.unl.pt">José Xavier</a>

Licensed under  <a href="http://creativecommons.org/licenses/by-sa/4.0/"
target="_blank"> <span style="color:#333399;font-size: 20px"> CC BY-SA 4.0  </span></a>
